docs/deprecations.md

```diff
@@ -6,6 +6,12 @@ layout: Doc
 
 # Serverless Framework Deprecations
 
+<a name="SERVICE_OBJECT_NOTATION"><div>&nbsp;</div></a>
+
+## `service` property object notation
+
+Starting from next major object notation for "service" property will no longer be recognized. Set "service" property directly with service name.
```

Comment:
In markdown let's put configuration property names in backticks not regular quotes..

Let's also write `Starting with v3.0.0` (note that this description is not meant to be exactly same as one we provider in a CLI. One in a CLI is supposed to be short informative, while here we can be more verbose)

Reply:
Changes done @medikoo